pkg/types/platform.go

```diff
@@ -45,6 +46,10 @@ func (e *Platform) GetCredentials() *Credentials {
 	return e.Credentials
 }
 
+func (e *Platform) GetPagingSequence() int {
```

Comment:
This looks like it could be generated but is not because of the notifications. Can't they be `Pageable` also and this can go in the base resource with this being generated. `ListObjects` is not implemented for `Notification` so it won't be called anyways